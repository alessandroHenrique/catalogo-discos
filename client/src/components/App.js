import React, { Component } from 'react';
import Header from './Header';

class App extends Component {
  render() {
    return (
      <div>
        <Header />
        <div className="container">
        </div>
      </div>
    );
  }
}

export default App;
