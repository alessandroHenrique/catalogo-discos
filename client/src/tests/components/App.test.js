import React from 'react';
import ReactDOM from 'react-dom';
import App from '../../components/App';
import Store from '../../store/configureStore';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

const store = Store();

it('renders without crashing', () => {
  const component = renderer.create(
    <Provider store={store}>
      <App />
    </Provider>
    );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
