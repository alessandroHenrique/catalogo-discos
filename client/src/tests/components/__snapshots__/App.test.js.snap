// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div
    className="header"
  >
    <div
      className="container"
    >
      <h1
        className="header__title"
      >
        Collection App
      </h1>
    </div>
  </div>
  <div
    className="container"
  >
    <form
      className="add-option"
      onSubmit={[Function]}
    >
      <input
        className="add-option__input"
        onChange={[Function]}
        type="text"
        value=""
      />
      <button
        className="button btn-secondary no-hover"
        disabled={true}
        type="submit"
      >
        Add Collection
      </button>
    </form>
    <ul
      className="list-group list-unstyled"
    />
  </div>
</div>
`;
